Test/ServerTelemetryChannel.Test/Shared.Tests/Implementation/TransmissionStorageTest.cs

```diff
@@ -529,7 +529,7 @@ public void LoadsTransmissionFromTheOldestTransmissionFileInTheFolder()
 
                 var oldestAddress = new Uri("http://oldest");
                 StubPlatformFile oldestFile = CreateTransmissionFile(oldestAddress);
-                oldestFile.OnGetDateCreated = () => DateTimeOffset.MinValue;
+                oldestFile.OnGetDateCreated = () => DateTimeOffset.Now.AddDays(-1);
```

Comment:
Sure, it is because now there is logic to check if files are older than two days, which means Breeze will reject them, and so test files can't be created with the min value for the creation date or they will be invalid and get deleted right away.